requirements.dev.txt

```diff
@@ -4,3 +4,7 @@ sphinx-autodoc-typehints
 recommonmark
 pypandoc
 twine
+
+pytest
```

Comment:
if I understand correctly `requirements.dev.txt` is for development, so pytest etc. should be declared there. Normally you don't run `setup.py` when import the project into IDE. Also, I am not certain if `setup.py` installs into virtualenvs.